src/commands/resumes/getResume.ts

```diff
@@ -0,0 +1,59 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { RESUME_EMBED_COLOUR } from '../../utils/embeds';
+
+class GetResumeCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'get-resume-command',
+      aliases: ['get-resume', 'resume'],
+      group: 'resumes',
+      memberName: 'get-resume',
+      description: 'Get an example resume!',
+      examples: [`${client.commandPrefix}resume web-dev 2`],
+      args: [
+        {
+          key: 'field',
+          prompt: 'The field of the resume (eg web-dev)',
+          type: 'string'
+        },
+        {
+          key: 'term',
+          prompt: 'The work-term the person with the resume will be in during their co-op (eg 3)',
+          type: 'integer'
+        }
+      ]
+    });
+  }
+
+  getResumeEmbed = (field: string, term: string): MessageEmbed => {
```

Comment:
To be consistent with rest of repo, i think getResumeEmbed should exist in it's own file, but the onRun remains.

Reply:
This is now done

Reply:
actually, in `suggestionsList.ts`, the embed is in the same file as the `onRun` command. I've put the `getResumeEmbed` command back into `getResume.ts` since you suggested to follow the same structure as `suggestions`, but let me know if I should do something different.